modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/Xref_update_metazoa_conf.pm

```diff
@@ -0,0 +1,64 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2017] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::Xref_update_metazoa_conf;
+
+use strict;
+use warnings;
+
+use base ('Bio::EnsEMBL::Production::Pipeline::PipeConfig::Xref_update_conf');
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+
+use Bio::EnsEMBL::ApiVersion qw/software_version/;
+
+
+sub default_options {
+    my ($self) = @_;
+
+    return {
+           %{ $self->SUPER::default_options() },
+           'email'            => $self->o('ENV', 'USER').'@ebi.ac.uk',
+           'release'          => software_version(),
+           'sql_dir'          => $self->o('ENV', 'HOME')."/work/lib/ensembl/misc-scripts/xref_mapping",
+           'pipeline_name'    => 'metazoa_xref_update_'.$self->o('release'),
+
+           ## 'job_factory' parameters
+           'species'          => ['caenorhabditis_elegans'],
+           'antispecies'      => [],
+           'division'         => [],
+           'taxon'            => 'wormbase',
+           'run_all'          => 0,
+           'force'            => ['caenorhabditis_elegans'],
+           'check_intentions' => 0,
+
+           ## Parameters for source download
+           'config_file'      => $self->o('ENV', 'HOME')."/work/lib/ensembl-production/modules/Bio/EnsEMBL/Production/Pipeline/Xrefs/xref_sources_metazoa.json",
+           'ftp_release'      => 'WS263',
```

Comment:
It seems dangerous to have a default value for ftp_release; presumably that's going to change frequently, so we'd need to update the conf file every release to keep in sync? It can be overwritten at instantiation, of course, but if you forget to do that, then you'll load old data by mistake, and maybe not even notice.

Reply:
The option ftp_release was provided so that the pipeline can s/WBRELEASE/WB265/ to once configured paths. I have not used it, I would rather not rely it:
  + I still need to change WormBase sources in `xref_sources.json` whenever WormBase has new species
  + I still need to edit the `xref_config.ini` to make the old code understand the species: we can have many species per taxon, and WormBase core species have a species-specific source